zookeeper/server/init.sls

```diff
@@ -7,7 +7,7 @@ include:
 
 move-zookeeper-dist-conf:
   cmd.run:
-    - name: mv {{ zk.real_home }}/conf {{ zk.real_config }}
+    - name: mv {{ zk.real_home }}/conf/* {{ zk.real_config }}/
```

Comment:
It seems we could replace `cmd.run` with [file.replace](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html#salt.states.file.rename) state function to rename conf directory into `zk.real_config`.
Also it should `require_in` the `zoo-cfg` file to make sure it would be created afterwards.

It would make things more clear and "stateful". What do you think?